typescript/src/languages/en-US/commands/starwars.json

```diff
@@ -0,0 +1,200 @@
+{
+    "filmDescription": "Get data for any given Star Wars movie",
+    "filmExtended": {
+        "usages": [
+            "Movie"
+        ],
+        "extendedHelp": "Uses a fuzzy search to match on both movie episode ID (1, 2...6) as well as movie title.",
+        "explainedUsage": [
+			[
+                "Movie",
+                "The movie for which you want to find data"
+			]
+		],
+        "examples": [
+			"1",
+			"hope",
+            "revenge of the sith"
+		],
+        "reminder": "At the moment only data on the first 6 movies is available. Data in movies 7 through 9 and spin-offs might be added in the future."
+    },
+    "filmQueryFail": "{{REDCROSS}} I am sorry, but that query failed. Are you sure `{{film}}` is actually a Star Wars movie?",
+    "filmEmbedTitles": {
+        "characters": "People in this film",
+        "creationDate": "Creation date",
+        "director": "Director",
+        "episodeId": "Episode ID",
+        "planets": "Planets in this film",
+        "producers": "Producers",
+        "releaseDate": "Release date",
+        "species": "Species in this film",
+        "starships": "Starships in this film",
+        "vehicles": "Vehicles in this film"
+    },
+    "peopleDescription": "Get data for any given character in Star Wars",
+    "peopleExtended": {
+        "usages": [
+            "Person"
+        ],
+        "extendedHelp": "Uses a fuzzy search on the name of the person",
+        "explainedUsage": [
+			[
```

Comment:
This file is a soup of spaces vs tabs.

Reply:
Fucking hell I blame that I removed those options from vscode settings that I re-added in this PR.